CHANGES.rst

```diff
@@ -1,21 +1,32 @@
 Changes
 =======
 
-Version 0.8.1 (UNRELEASED)
+Version 0.8.1 (2022-02-09)
 --------------------------
 
 - Users:
     - Adds support for specifying ``kubernetes_job_timeout`` for Kubernetes compute backend jobs.
+    - Adds Kubernetes job memory limits validation before publishing workflow submission.
+    - Adds support for HTML preview of workspace files in the user interface.
```

Comment:
"... in the web user interface" just to make it fully clear. (There can be command-line user interface, terminal user interface, web user interface...)